### PR TITLE
Create ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by jdre-c8y

Ticket: [MTM-60471](https://cumulocity.atlassian.net/browse/MTM-60471)
Original PR: [7206](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7206)

[MTM-60471]: https://cumulocity.atlassian.net/browse/MTM-60471?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device name goes missing when switching to smart rule tab (#7206)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]: The Claude AI suggested to use the following title:
```suggestion
title: Device name missing when switching to smart rule tab
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60471
+version: 1021.0.0
+---
+Device name goes missing when switching to smart rule tab (#7206)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]: The Claude AI suggested to use the following body:
```suggestion
In the device management application, when viewing the details of a device and switching to the smart rule tab, the device name was previously not displayed at the top of the page anymore. This issue has been resolved and the device name now correctly remains visible when navigating to the smart rule tab for a device. The fix ensures that the device name is properly retained and displayed across all tabs within the device details view, providing a consistent user experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md (hunk 2 above)
Comment (jdre-c8y) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]:
```suggestion
Device name in page title was missing when switching to 'Smart rules' tab. It is displayed properly now.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60471
+version: 1021.0.0
+---
+Device name in page title was missing when switching to 'Smart rules' tab. It is displayed properly now.
```
Comment (jdre-c8y) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]:
```suggestion
Device name in Cockpit page title was missing when switching to 'Smart rules' tab. It is displayed properly now.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60471
+version: 1021.0.0
+---
+Device name in Cockpit page title was missing when switching to 'Smart rules' tab. It is displayed properly now.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]:
```suggestion
In the Cockpit application, when switching to the **Smart rules** tab of a device in a group, the device name was missing in the page title. This issue has been fixed and the device name is displayed properly now.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Device name missing when switching to smart rule tab
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-device-name-goes-missing-when-switching-to-smart-rule-tab-7206.md]:
```suggestion
title: Device name displayed properly when switching to the Smart rules tab
```

**Review by BeateRixen — APPROVED** [review]

**Comment by jdre-c8y** [comment]
@rahultsag  can you have a look?

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**